features/facebookImport/test/utils/analyses-execution.js

```diff
@@ -17,10 +16,11 @@ export async function runAnalysisForExport(
     zipFile,
     pod = new MockPod()
 ) {
-    const facebookAccount = await importZip({
-        dataImporters,
+    const facebookAccount = new FacebookAccount();
```

Comment:
Have you thought about an alternative to this two-step process? If the account is not created from an importer file, when and where is it going to be created?

Reply:
I don't understand this - I didn't really get to the analyses yet at all neither to their tests. Just changed this to keep to stable

Reply:
OK, just asking. Before, since the facebookAccount got set when going through the different importers, it needed to be created empty. That's not needed any more.
It was a side question, anyway.